index_ko.html

```diff
@@ -4,21 +4,22 @@
 subtitle: OS X 용 패키지 관리자
 
 pagecontent:
-  what: Homebrew는 Apple에서 제공하지 않지만 <a href="https://github.com/Homebrew/homebrew/tree/master/Library/Formula" title="List of Homebrew packages">필요한 패키지</a>를 설치합니다.
-  how: Homebrew는 자신의 디렉토리에 패키지를 설치하고, <code>/usr/local</code>안에 그 파일을 심볼링 링크로 겁니다.
-  prefix: Homebrew는 prefix가 붙지 않는 파일은 설치 하지 않지만, 여러분이 원하는 위치 어느곳이든 설치할 수 있습니다.
-  createpackages: 자신의 Homebrew 패키지를 간편하게 만들 수 있습니다.
-  hack: Homebrew는 모두 git과 ruby기반으로 되어 있어, 여러분이 이미 가진 지식으로 쉽게 수정하거나 업데이트, 병합할 수 있습니다.
+  what: Homebrew는 Apple에서 제공하지 않는 <a href="https://github.com/Homebrew/homebrew/tree/master/Library/Formula" title="List of Homebrew packages">유용한 패키지 관리자</a>를 설치합니다.
+  how: Homebrew는 전용 디렉토리에 패키지를 설치하고 <code>/usr/local</code> 위치로 심볼릭 링크를 연결합니다.
+  prefix: Homebrew는 전용 디렉토리 외부에 설치하는 일은 없지만 원하는 위치에 설치할 수도 있습니다.
+  createpackages: 간단하게 Homebrew 패키지를 만들어 보세요.
+  hack: 모두 git과 ruby 기반으로 되어 있으며, 쉽게 수정을 되돌리거나 원본 코드의 수정사항을 병합하면서 해킹하세요.
   formula: "Homebrew formula는 간단한 Ruby 스크립트입니다:"
-  complement: Homebrew는 OS X을 보완해 줍니다. <code>gem</code>명령으로 gem을, <code>brew</code>명령으로 gem의 의존관계의 모듈을 설치합니다.
+  complement: Homebrew는 OS X 개발 환경을 개선합니다. <code>gem</code> 명령으로 gem을, <code>brew</code> 명령으로 gem의 의존성 모듈을 설치하세요.
   install:
     install: Homebrew 설치하기
-    paste: 터미널 프롬프트에서 붙여넣기 하세요.
-    what: 스크립트에서 무엇을 하는지에 대해 설명하고, 실행하기 전에 일단 정지합니다. 더 자세한 설치 관련사항을 보려면  <a href='https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Installation.md#installation'>여기</a>를 봐 주세요 (10.5 버전 필요).
+    paste: 터미널에 붙여넣기 하세요.
+    what: 해당 스크립트는 무엇을 할지 설명하고 실행하기 전에 잠시 멈춥니다. 더 자세한 설치 관련사항을 보려면 <a href='https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Installation.md#installation'>여기</a>를 참고하세요 (10.5 버전 필요).
   doc:
     further: 추가 문서
   foot:
     code: Original code by <a href="http://methylblue.com/">Max Howell</a>.
     page: Website by <a href="http://exomel.com">Rémi Prévost</a>.
     translation: Translation by <a href="http://rkjun.wordpress.com/">Juntai Park</a>.
+    translation: Translation by <a href="https://github.com/ghmlee">Graham Lee</a>.
```

Comment:
Add this to the previous line; as-is this will stop Juntai from getting credit. Thanks!